Products/CMFPlone/__init__.py

```diff
@@ -10,8 +10,10 @@
 __version__ = pkg_resources.require("Products.CMFPlone")[0].version
 
 if __version__ < '7':
-    from Products.CMFCore.explicitacquisition import PTA_ENV_KEY
+    from Products.CMFCore import explicitacquisition
+    PTA_ENV_KEY = explicitacquisition.PTA_ENV_KEY
     os.environ[PTA_ENV_KEY] = os.environ.get(PTA_ENV_KEY, 'false')
```

Comment:
I'd say we can remove this line; it's less confusing if we don't mutate the environment.

```suggestion
```

Reply:
I have rebased and force-pushed the branch, and copied over the code that is already in use on master.  It has some comments that explain a bit more what we are actually doing here.